src/smc-webapp/editor_chat.cjsx

```diff
@@ -799,10 +896,16 @@ ChatRoom = (name) -> rclass
         #debounces it so that the preview shows up then calls
         @scroll_to_bottom()
 
+    send_video_id: ->
+        if not @has_video_id()[0] and not @props.is_side_chat
+            @_video_chat_id = misc.uuid()
+            @props.actions.send_video_chat("Video Chat Room ID is: #{@_video_chat_id}")
+
     componentWillMount: ->
         @set_preview_state = underscore.debounce(@set_preview_state, 500)
         @set_chat_log_state = underscore.debounce(@set_chat_log_state, 10)
         @debounce_bottom = underscore.debounce(@debounce_bottom, 10)
+        @send_video_id = underscore.debounce(@send_video_id, 10)
```

Comment:
I'm just reading the patch to see what's going on. Here, I see it is debouching to 10ms only? My feeling is, that this is still too fast.